ZUGFeRD/InvoiceDescriptor.cs

```diff
@@ -295,6 +295,23 @@ public class InvoiceDescriptor
         public DateTime? BillingPeriodEnd { get; set; }
 
 
+        /// <summary>
+        /// Spezification Identifier
+        /// </summary>
+        public string SpecificationId { get; set; }
```

Comment:
I make the setter internal because you already set the Specification ID (BT24) by using method "Profile.EnumToString()". But until now I have no possibility to access it outside. Therefore I added this new proptery "SpecificationID".
Now this property is set when reading and saving an invoice.

Reply:
I still don't get why this Id is necessary in parallel to the profile.

Reply:
Because in an existing project it is not sufficient to know only the profile. In this project currently the Specification ID is used to make some decisions how the business process will go on.
Currently it's not possible to get the version of the invoice.